ACE/ace/INET_Addr.cpp

```diff
@@ -371,8 +371,11 @@ ACE_INET_Addr::set (u_short port_number,
 #endif /* ACE_HAS_IPV6 && ACE_USES_IPV4_IPV6_MIGRATION */
 
 #ifdef ACE_HAS_IPV6
+
+#ifndef AIX
```

Comment:
I don't access to any AIX machines, the only thing I know is that AIX getaddrinfo() seems to behave differently based on the scoreboard results in the past. I just need to see if the ifdef can help on AIX.